pandas/tests/test_nanops.py

```diff
@@ -300,33 +270,20 @@ def _badobj_wrap(self, value, func, allow_complex=True, **kwargs):
                 value = value.astype("f8")
         return func(value, **kwargs)
 
-    def test_nanany(self):
-        self.check_funs(
-            nanops.nanany,
-            np.any,
-            allow_all_nan=False,
-            allow_str=False,
-            allow_date=False,
-            allow_tdelta=False,
-        )
-
-    def test_nanall(self):
+    @pytest.mark.parametrize(
+        "nan_op,np_op", [(nanops.nanany, np.any), (nanops.nanall, np.all)]
+    )
+    def test_nan_funcs(self, nan_op, np_op):
+        # TODO: allow tdelta, doesn't break tests
         self.check_funs(
-            nanops.nanall,
-            np.all,
-            allow_all_nan=False,
-            allow_str=False,
-            allow_date=False,
-            allow_tdelta=False,
+            nan_op, np_op, allow_all_nan=False, allow_date=False, allow_tdelta=False
         )
 
     def test_nansum(self):
         self.check_funs(
             nanops.nansum,
             np.sum,
-            allow_str=False,
```

Comment:
Is this just universally removed here?

Reply:
yah, so just removed allow_str altogether.